### PR TITLE
Add getWebookStrategy()

### PR DESCRIPTION
**Pull request by minz1027**
- API will use getWebookStrategy() to set auth strategy for the webhook

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -219,6 +219,20 @@ A configuration that can be passed to the [bell][bell] OAuth module to authentic
 #### Expected Promise Response
 1. Resolve with a valid [bell][bell] configuration
 
+### getWebhookStrategy
+
+#### Expected Outcome
```
Comment (d2lam) [inline README.md]: Should mention here that it will return `null` if no auth strategy is needed.

Comment (minz1027) [inline README.md]: Fixed

File: README.md [hunk 2]
```diff
@@ -219,6 +219,21 @@ A configuration that can be passed to the [bell][bell] OAuth module to authentic
...
+A hapi webhook auth strategy object that can be passed to the hapi server.
```
Comment (stjohnjohnson) [inline README.md]: Wouldn't it be easier to just implement the ~10 lines of validation required https://github.com/mhazy/hapi-github-webhooks/blob/master/lib/hmac.js#L11-L17 by passing in the entire headers to the parseHook function.

Comment (minz1027) [inline README.md]: So the parseHook() should  take care of the auth for the webhook? Then we need to make it clear in the base that the parseHook is responsible for that. 
Can I import and re-use this? https://github.com/mhazy/hapi-github-webhooks/blob/master/lib/hmac.js#L11-L17

Comment (tkyi) [inline README.md]: Or did you mean to just copy those ~10 lines over?

**Comment by tkyi** [comment]
Closing in favor of screwdriver-cd/scm-github#29

**Pull request closed without merging**